openslides/mediafiles/static/js/mediafiles/site.js

```diff
@@ -111,6 +126,111 @@ angular.module('OpenSlidesApp.mediafiles.site', ['ngFileUpload', 'OpenSlidesApp.
         $scope.delete = function (mediafile) {
             Mediafile.destroy(mediafile.id);
         };
+
+        // ** PDF presentation functions **/
+        // show document on projector
+        $scope.showPdf = function (mediafile) {
+            var postUrl,
+                data;
+            if($scope.presentedMediafiles.length > 0) {
+                // update first mediafile, at the moment there should not be more
+                var uuid = $scope.presentedMediafiles[0].uuid;
+                postUrl = '/rest/core/projector/1/update_elements/';
+                data = {};
+                data[uuid] = {
+                    id: mediafile.id,
+                    numPages: mediafile.mediafile.pages,
+                    page: 1,
+                    pageFit: true,
+                    scale: 1,
+                    visible: true
+                };
+            } else {
+                postUrl = '/rest/core/projector/1/prune_elements/';
+                data = [{
+                    name: 'mediafiles/mediafile',
+                    id: mediafile.id,
+                    numPages: mediafile.mediafile.pages,
+                    visible: true,
+                    pageFit: true,
+                    scale: 1,
+                    page: 1
+                }];
+            }
+            $http.post(postUrl, data);
+        };
+
+        function sendMediafileCommand(data) {
+            var mediafileElement = getCurrentlyPresentedMediafile();
+            var updateData = _.extend({}, mediafileElement);
+            _.extend(updateData, data);
+            var postData = {};
+            postData[mediafileElement.uuid] = updateData;
+            $http.post('/rest/core/projector/1/update_elements/', postData);
+        }
+
+        function getCurrentlyPresentedMediafile() {
+            return $scope.presentedMediafiles[0];
+        }
+
+        $scope.mediafileGoPrevious = function () {
+            var mediafileElement = getCurrentlyPresentedMediafile();
+            if (mediafileElement.page <= 1) {
```

Comment:
Proposal: Switch the logic: if ...page > 1 then sendCommand. Thats all. No empty return in the middle of the function neccessary.